internal/mithril-persistence/src/database/repository/cardano_transaction_repository.rs

```diff
@@ -182,16 +203,12 @@ impl CardanoTransactionRepository {
     ) -> StdResult<()> {
         const DB_TRANSACTION_SIZE: usize = 100000;
         for transactions_in_db_transaction_chunk in transactions.chunks(DB_TRANSACTION_SIZE) {
-            let transaction = self.connection.begin_transaction()?;
-
             // Chunk transactions to avoid an error when we exceed sqlite binding limitations
             for transactions_in_chunk in transactions_in_db_transaction_chunk.chunks(100) {
                 self.create_transactions(transactions_in_chunk.to_vec())
                     .await
                     .with_context(|| "CardanoTransactionRepository can not store transactions")?;
             }
-
-            transaction.commit()?;
```

Comment:
Is that really what we want ? Making the transaction wrap only one query means that it doesn't do anything more than SQLite automatic transaction, making us loose the performance gain of transactions entirely.